website/src/components/Edit/SequencesForm.tsx

```diff
@@ -126,6 +133,28 @@ export const SequencesForm: FC<SequenceFormProps> = ({ editableSequences, setEdi
                                     : undefined
                             }
                             showUndo={true}
+                            onDownload={
```

Comment:
Is it possible to keep this code in a function generateAndDownloadFastaFile? - it might be easier to read

Reply:
Thanks - now done!